htdocs/modulebuilder/index.php

```diff
@@ -2052,7 +2075,7 @@ function getLicenceHeader($user, $langs, $now)
 			'sql/llx_mymodule_myobject_extrafields.key.sql' => 'sql/llx_'.strtolower($module).'_'.strtolower($objectname).'_extrafields.key.sql',
 			'scripts/myobject.php' => 'scripts/'.strtolower($objectname).'.php',
 			'class/myobject.class.php' => 'class/'.strtolower($objectname).'.class.php',
-			'class/api_myobject.class.php' => 'class/api_'.strtolower($module).'.class.php',
+			'class/api_myobject.class.php' => strtolower($module),
```

Comment:
Why changing this line ?

Reply:
The PR generates warning in CI. But it is not possible to fix them because your PR was not created with permission to others to contributes.

Reply:
it's an error sorry

Reply:
```suggestion
			'class/api_myobject.class.php' => 'class/api_'.strtolower($module).'.class.php',
```

Reply:
> The PR generates warning in CI. But it is not possible to fix them because your PR was not created with permission to others to contributes.

I enabled "Allow edits and access to secrets by maintainers"

Reply:
I confirm pr is open for fixing now, it will sabe time on both side, thanks....